src/components/authenticationPage.js

```diff
@@ -0,0 +1,44 @@
+import React, {Component} from 'react';
+
+class WaitingPage extends Component {
+    
+async postData(url = '', data = {},token) {
+
+    const response = await fetch(url, {
+      method: 'POST', 
+      headers: {
+        'Content-Type': 'application/json',
+        'Authorisation':token
+      },
+      redirect: 'follow', // manual, *follow, error
+      referrerPolicy: 'no-referrer', // no-referrer, *no-referrer-when-downgrade, origin, origin-when-cross-origin, same-origin, strict-origin, strict-origin-when-cross-origin, unsafe-url
+      body: JSON.stringify(data) // body data type must match "Content-Type" header
+      });
+    return response.json(); // parses JSON response into native JavaScript objects
+    }
+  
+    getUrlVars = (vars) =>   {
+        window.location.href.replace(/[?&]+([^=&]+)=([^&]*)/gi, function(m,key,value) {
+        vars[key] = value;
+    });
+    
+    }
+    render() {
+        let vars = {};
+        this.getUrlVars(vars)
+        this.postData('http://localhost:8080/connect', {}, vars.access_token)
+        .then((data) => {
+        console.log(data); // JSON data parsed by `response.json()` call
```

Comment:
at the moment as the orchestration service isn't return what it should i think we should leave it in to make it easier when its done? but happy to remove if you this otherwise